### DISCUSSION TITLE
移除直播间相同弹幕连续提示模块

### DISCUSSION THREAD

**Discussion by andywang425** [andywang425/BLTH#486] [Development]
B站好像没有相同弹幕连续提示的功能了？弹窗对应的css是`#combo-card`，而`combo-card`在代码里已经找不到了。@ADJazzzz

![](https://private-user-images.githubusercontent.com/68748165/300242870-cc3344c6-d180-4901-9f3c-aa0997de9824.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDcyMDY2MjksIm5iZiI6MTcwNzIwNjMyOSwicGF0aCI6Ii82ODc0ODE2NS8zMDAyNDI4NzAtY2MzMzQ0YzYtZDE4MC00OTAxLTlmM2MtYWEwOTk3ZGU5ODI0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjA2VDA3NTg0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRmMTZjN2RiOGM2NDRiZmVmOTY1NjZjNjVhNjY4YTQ1NjdjNTJjMGVhMmY5ODhjZmVmZGFhNTMxYTcwYzgwMzgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.uYJdeGB01xtrWYiVGYiGFbdpbwTWQjX_WNLoo529fGQ)

**Comment by ADJazzzz** [comment]
它是检查到要相同弹幕的时候才会注入，和幻星派对一样

**Reply by andywang425** [reply]
幻星派对的css字符串我在js里找到了，但没找到这个。

**Reply by andywang425** [reply]
看直播的时候我也没见过相同弹幕连续提示的弹窗

**Reply by ADJazzzz** [reply]
> 幻星派对的css字符串我在js里找到了，但没找到这个。

这个可能b站退回会版本了，现在网页端、手机端都没有了
昨天凌晨看直播的时候还在的

**Reply by ADJazzzz** [reply]
~~我建议是先留着，如果b站确实是把它删除了那就可以从代码移除这个功能了~~

它又出来了
![{A55F1377-E164-4cb6-961A-C2ECC45B9FA7}](https://github.com/andywang425/BLTH/assets/68748165/ce060a0d-89f1-4ffd-af13-8c28c07cddac)

**Reply by andywang425** [reply]
嗯，留着吧，之后我把移除幻星派对弄回来。

**Reply by andywang425** [reply]
这个功能还不错，有时候重复弹幕一片一片的。

**Comment by andywang425** [comment]
我现在有点明白了，有时候js文件是动态获取并插入到DOM执行的。这应该是Vue的模块懒加载功能。
